### ISSUE TITLE
How to start container?

### ISSUE THREAD

**Issue by Alex-1357** [dotnet/Docker.DotNet#551]
Unfortunately, this code

        Dim Env = New List(Of String)({"RPC_PASSWORD=$(xxd -l 32 -c 32 -p /dev/urandom)",
                                       "EXTERNAL_IP='157XX.XX.85'",
                                       "MASTERNODE_PRIV_KEY='41kWXqrknGeLLEqnUgRBPtrBxQ3o5VBp4bkRy4fHN3ZpzURHdkr'",
                                       "TX_INDEX='1'",
                                       "SEED_NODE='95.XX.XX.73'",
                                       "PORT='46004'",
                                       "RPC_ALLOW_IP='127.0.0.1'",
                                       "RPC_CONNECT_IP='127.0.0.1'"})
        Dim ExposedPorts = New Dictionary(Of String, EmptyStruct)
        ExposedPorts.Add("'157.XX.XX.85:46004:46004'", New EmptyStruct)
        'Dim Labels = New Dictionary(Of String, String)({"Api test"})
        Dim Prm = New CreateContainerParameters With {
                      .Image = "alexev275/digiwage:latest",
                      .Name = $"digiwage.{Guid.NewGuid}",
                      .Env = Env,
                      .ExposedPorts = ExposedPorts,
                      .Tty = True
        }
        Dim CreateContainer As CreateContainerResponse = DockerHub.Containers.CreateContainerAsync(Prm).Result
        Dim StartResTask As Task(Of Boolean) = DockerHub.Containers.StartContainerAsync(CreateContainer.ID, New ContainerStartParameters())
        StartResTask.Wait()

produced only template on container without starting it
![docker_01282022_001148-1](https://user-images.githubusercontent.com/86534217/151455902-e9076096-59c1-40ca-a1cb-488b39e9a2c0.png)

and status RunToCompletion, but container don't started

![docker_01282022_001500-1](https://user-images.githubusercontent.com/86534217/151455971-d7c1f522-2c49-476b-a637-c1779500f11f.png)

How to start container?

**Comment by OneCricketeer** [comment]
Whatever `startup.sh` does, it appears to terminate with ExitCode=1

**Comment by Alex-1357** [comment]
Yes, thank you, I found a reason, RPC_PASSWORD is not a valid string for net core. However, I don't understand how I can understand successful or unsuccessful container start. Of course, log allow something, but log working bad, I can not see last 5-10 string in successfully started container, I see it only when container stopped.

**Comment by OneCricketeer** [comment]
The `Status` field should say the container is "up" or, as the debugger image shows "RanToCompletion" (which seems to use `1` as the error code, indicating an internal process failure in your startup script). A `0` error code would mean you entrypoint had terminated successfully
